### PR TITLE
Added patched VPA recommender with memory-saver mode enabled

### PR DESCRIPTION
**Pull request by arjunrn**
Added patched version of the VPA recommender with memory-saver mode enabled.

### CODE REVIEW

**Comment by mikkeloscar** [comment]
It's failing with `OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"--stderrthresholdinfo=info\": executable file not found in $PATH": unknown
` fwiw.

**Comment by arjunrn** [comment]
:+1:

File: cluster/manifests/01-vertical-pod-autoscaler/recommender-deployment.yaml [hunk 1]
```diff
@@ -16,13 +16,19 @@ spec:
...
       containers:
       - name: recommender
-        image: registry.opensource.zalan.do/teapot/vpa-recommender:patched-0.3.0-master-9
+        image: registry.opensource.zalan.do/teapot/vpa-recommender:patched-0.3.0-master-12
+        args:
+        - --stderrthreshold=info
+        - --v=5
+        - --memory-saver
+        command:
+        - "./recommender"
```
Comment (szuecs) [inline cluster/manifests/01-vertical-pod-autoscaler/recommender-deployment.yaml]: is the "." in front correct? 
It now depends on CWD and I would like to have it better absolute path instead.

Comment (arjunrn) [reply cluster/manifests/01-vertical-pod-autoscaler/recommender-deployment.yaml]: Made the change

**Comment by mikkeloscar** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**